pkg/controller/schedule_controller.go

```diff
@@ -142,6 +142,15 @@ func parseCronSchedule(itm *velerov1.Schedule, logger logrus.FieldLogger) (cron.
 		return nil, validationErrors
 	}
 
+	// For new created schedules, DefaultVolumesToRestic is not allowed
+	// TODO: post v1.10. Remove this code block after DefaultVolumesToRestic is removed from CRD
+	if itm.Status.Phase != velerov1.SchedulePhaseEnabled {
```

Comment:
The `parseCronSchedule` runs not only for the newly created schedules, so this will cause the old schedule failure

Reply:
And preventing the usage of `DefaultVolumesToRestic` is a break change which should introduce a new version of CRD

Reply:
After a discussion, the conclusion is as below:

- We will not change the CRD's version for now. We will accumulate more changes of the CRD and change the version for all, we will also add mutation webhook.
- We will not block the newly created schedule CR's if their DefaultVolumesToRestic field is set to true
- Instead, we add a warning log when we find the to-be-deprecated field is used